CIPs/CIP-85/CIP-85.md

```diff
@@ -17,46 +17,50 @@ Provide schemas and associated tools to help interact with large lists in Cerami
 
 ## Abstract
 
-This CIP presents a way to store a possibly large list of items split into multiple documents based on two complementary schemas: `AppendCollection` and `CollectionSlice`.
-This allows to create a virtually infinite list of items such as a feed by leveraging multiple Ceramic documents rather than a single one.
+This CIP presents a way to store a possibly large list of items split into multiple streams based on two complementary schemas: `AppendCollection` and `CollectionSlice`.
+This allows to create a virtually infinite list of items such as a feed by leveraging multiple Ceramic streams rather than a single one.
 
 ## Motivation
 
-Storing a list of items in a single Ceramic document can make the document grow large in size, possibly exceeding Ceramic's internal limits.
+Storing a list of items in a single Ceramic stream can make the stream grow large in size, possibly exceeding Ceramic's internal limits.
 Providing a standard way to represent large lists such as feeds would be useful to provide reference schemas to solve this issue and associated tools to simplify interactions.
 
 ## Specification
 
 ### Concepts
 
-#### Doc references
+#### Stream references
 
-The schemas use document references defined in [CIP-82](https://github.com/ceramicnetwork/CIP/issues/82) to identify relations.
+The schemas use stream references defined in [CIP-82](https://github.com/ceramicnetwork/CIP/issues/82) to identify relations.
 
 #### Cursors
 
-A cursor is a unique identifier for an item in the collection, made of the tuple (CollectionSlice DocID, item index).
+A cursor is a unique identifier for an item in the collection, made of the tuple (CollectionSlice StreamID, item index).
 A collection slice should contain a maximum of 256 items, so the index of an item in the slice can be represented in a single byte.
 
+#### Slice tag
```

Comment:
Hmm... I wonder if it makes sense to use `tags` in this way, or if we should wait for https://github.com/ceramicnetwork/js-ceramic/issues/1342 and use a more semantically meaningful object structure in the metadata for this

Reply:
Sure, we should use whatever makes most sense for Ceramic, I wasn't aware there would be a new field in the metadata for this.
Might be good to clarify the relations and differences between `family`, `tags` and this new field i guess, as I thought `family` was the field meant for indexing (at least in the context of IDX).

Reply:
I think the idea is for both family and tags to be used by indexers

Reply:
my vote would be to do this with the new metadata object from https://github.com/ceramicnetwork/js-ceramic/issues/1342 so the field names could have more semantic meaning, but I don't feel super strongly and could be convinced it makes sense to just do this with tags

Reply:
`family` is the primary key used for indexing. I agree with @stbrody's point unless we actually want the indexer to more easily find it using the `tag`.